src/EditorFeatures/CSharpTest2/Recommendations/AbstractKeywordRecommenderTests.cs

```diff
@@ -162,7 +162,7 @@ public async Task TestNotBeforeUsing()
 using Goo;");
         }
 
-        [WpfFact(Skip = "https://github.com/dotnet/roslyn/issues/9880")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/9880")]
```

Comment:
looksl ike we likely switched things to Fact previously, but we didn't miss all these cases since the find/replace was too restrictive and didn't understand Skip.